lib/auth/home.dart

```diff
@@ -6,17 +6,13 @@ import 'dart:io';
 import 'dart:async';
 import 'dart:developer';
 
-import 'package:vaultSDK/udPointCloud.dart';
-import 'package:vaultSDK/udContext.dart';
-
 import '../widgets/ErrorMsg.dart';
+import '../auth/SceneViewer.dart';
```

Comment:
might be case sensitive. 
```suggestion
import '../auth/sceneViewer.dart';
```

Reply:
Looks like windows has a case insensitive file system.